lib/MusicBrainz/Server/Data/Collection.pm

```diff
@@ -89,6 +89,15 @@ sub contains_entity {
         $collection_id, $id) ? 1 : 0;
 }
 
+sub non_empty {
```

Comment:
For series we use `is_empty`, so I can do that for consistency. I guess I inverted this one based on how it was being used.